iothub/device/src/TransportSettings/IotHubClientAmqpSettings.cs

```diff
@@ -78,6 +94,7 @@ internal override IotHubClientTransportSettings Clone()
                 PrefetchCount = PrefetchCount,
                 RemoteCertificateValidationCallback = RemoteCertificateValidationCallback,
                 ConnectionPoolSettings = ConnectionPoolSettings,
+                ClientWebSocket = ClientWebSocket,
```

Comment:
Will the test need to be updated to have a value here to validate?

Reply:
The test does not necessarily need to be updated but I have still updated it, just to be on the safe,